DIRECTORY.md

```diff
@@ -42,6 +42,7 @@
 
 ## Sorting
   * [Bogo Sort](https://github.com/TheAlgorithms/Ruby/blob/master/sorting/bogo_sort.rb)
+  * [Bogo Sort Test](https://github.com/TheAlgorithms/Ruby/blob/master/sorting/bogo_sort_test.rb)
```

Comment:
Could we consider merging `Bogo Sort` + `Bogo Sort Test` in the same directory and have only 1 item in the `DIRECTORY.md`?

Reply:
Checking the other repos, I noticed that we don't do this, so I'm just removing and merging. Thanks.

Reply:
Thanks for comments.

It seems that GitHub Action has automatically added `Bogo Sort Test` again on 687840f22f6dab1a2b08395e8df39630099660f2. Maybe we should consider update [the workflow](https://github.com/TheAlgorithms/Ruby/blob/687840f22f6dab1a2b08395e8df39630099660f2/.github/workflows/update_directory_md.yml) to skip `*_test.rb` in building `DIRECTORY.md`.

Let me know if you have any feedback about testing (opening a new issue, probably).